go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/banzaicloud/istio-client-go v0.0.17
 	github.com/banzaicloud/istio-operator/api/v2 v2.15.1
 	github.com/banzaicloud/k8s-objectmatcher v1.8.0
-	github.com/banzaicloud/koperator/api v0.28.7
+	github.com/banzaicloud/koperator/api v0.28.8
```

Comment:
**Required**: We **MUST** update it [here](https://github.com/banzaicloud/koperator/blob/master/tests/e2e/go.mod#L9) as well (and `go mod tidy` afterwards).

Reply:
Thanks, done!